### PR TITLE
Edition statistics

### PR DESCRIPTION
**Pull request by mariuscoto**

### CODE REVIEW

**Comment by mariuscoto** [comment]
@maria please have a look

File: config/models/activity.js [hunk 1]
```diff
@@ -13,7 +13,13 @@ var editionSchema = mongoose.Schema({
   link         : String,
   people       : [String], // Saved as NAME:ROLE
   start        : Date,
-  end          : Date
+  end          : Date,
+  enrolments   : Number,
+  participants : Number,
+  prizes       : Number,
+  budget       : Number,
```
Comment (maria) [inline config/models/activity.js]: I don't agree with budget and I thought we said no it. It's an internal stuff and it doesn't represents the value of the project.

File: routes/activities.js [hunk 2]
```diff
@@ -174,7 +231,7 @@ exports.edition = function(req, res) {
 
     res.render('edition', {
       'activity' : _self.activity,
-      'edition'  : _self.edition,
+      'myedition': _self.edition,
```
Comment (maria) [inline routes/activities.js]: `myedition` is the current edition added/edited?

File: utils.js [hunk 3]
```diff
@@ -32,4 +34,47 @@ exports.mail_community = function(new_event) {
...
+exports.setFormattedDateForEdit = function(datestr) {
+  return new Date(
+    datestr.substring(6,10),
+    parseInt(datestr.substring(3,5)) - 1,
+    datestr.substring(0,2)
+  )
+}
+
+
+
```
Comment (maria) [inline utils.js]: one blank line.. you can setup sublime to do this for you.

File: views/activity.jade [hunk 4]
```diff
@@ -9,27 +9,70 @@ block content
...
+            value=(edition? edition.end_format:false))
+      .row
+        .form-group.col-lg-12
+          h3 Statistics (can be filled in later)
+          label Number of people that signed up
+          input(
+            name='enrolments', type='text',
+            value=(edition? edition.enrolments:false))
+          br
+          label Number of people that actualy participated
```
Comment (maria) [inline views/activity.jade]: `actually`

File: views/activity.jade [hunk 5]
```diff
@@ -9,27 +9,70 @@ block content
...
           label End time
           br
-          input.datepicker(name='end_date', type='text', required)
+          input.datepicker(
+            name='end_date', type='text', required
+            value=(edition? edition.end_format:false))
+      .row
+        .form-group.col-lg-12
+          h3 Statistics (can be filled in later)
+          label Number of people that signed up
```
Comment (maria) [inline views/activity.jade]: I thin `who` is better than `that` when talking about persons.

File: views/activity.jade [hunk 6]
```diff
@@ -9,27 +9,70 @@ block content
...
+          input(
+            name='enrolments', type='text',
+            value=(edition? edition.enrolments:false))
+          br
+          label Number of people that actualy participated
+          input(
+            name='participants', type='text',
+            value=(edition? edition.participants:false))
+          br
+          label Number of prizes offered
```
Comment (maria) [inline views/activity.jade]: `given` or something else is better than `offered`, is like you put them there and let the participants grab them.

File: views/activity.jade [hunk 7]
```diff
@@ -9,27 +9,70 @@ block content
...
+          input(
+            name='prizes', type='text',
+            value=(edition? edition.prizes:false))
+          br
+          label Budget used for this edition
+          input(
+            name='budget', type='text',
+            value=(edition? edition.budget:false))
+          br
+          label Number of projects involved
```
Comment (maria) [inline views/activity.jade]: `contributed at` > `involved`

**Comment by maria** [comment]
code review done. app review - I can't login on my localhost I get an error from Google Auth app.

**Pull request merged**